src/tests/ftest/cart/no_pmix/cart_nopmix_multictx_one_node.py

```diff
@@ -59,7 +59,7 @@ def test_cart_no_pmix(self):
         test_env = self.pass_env
         p = subprocess.Popen([cmd], env=test_env, stdout=subprocess.PIPE)
 
-        rc = self.utils.wait_process(p, 10)
+        rc = self.utils.wait_process(p, 30)
```

Comment:
How do we know that 30 is enough?  What happens when we need 40 next month? Is there any other method other than increasing the time out to resolve this issue?

Reply:
This is a maximum timeout that is set for the test; if test does not finish executing within that timeframe we fail the test. 
Value of 10 was really close to the amount of time that test took to run (it takes between 8 and 9.5 seconds on most runs) leading to sporadic failures when test exceeded 10 second threshold. 

If the value of 30 wont be sufficient and we fail again then that would indicate a major performance regression and would need to be investigated and fixed instead of time being bumped again.